CMakeLists.txt

```diff
@@ -97,6 +97,16 @@ endif()
 
 project(wxWidgets VERSION ${wxVERSION} LANGUAGES ${wxLANGUAGES})
 
+if (WIN32 AND "${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
+    # by default msys2 convert option  /implib:../myLib.lib  incorrect as implib;myLib.lib 
+    message(STATUS " !!! enventory variable MSYS2_ARG_CONV_EXCL is: [$ENV{MSYS2_ARG_CONV_EXCL}] mast by [/implib:] see https://www.msys2.org/docs/filesystem-paths/" COMMENT "set MSYS2_ARG_CONV_EXCL=/implib" )
```

Comment:
Without this code: 
```
set(CMAKE_RC_COMPILE_OBJECT "<CMAKE_RC_COMPILER>  <DEFINES> -i ${CMAKE_SOURCE_DIR}/include -i ${CMAKE_SOURCE_DIR}/samples -fo <OBJECT> <SOURCE>")
```
I have built Error: error: no such file or directory: 'version.rc.res', but in build dir I see  "version.rc.res.res." instead of "version.rc.res"  If I manually rename "version.rc.res.res." to "version.rc.res", build Will be successful.

It reproduce in our projects too. I think, it is CMake+LLVM bug. We are create issue  https://gitlab.kitware.com/cmake/cmake/-/issues/24824, but

I'm not sure that waiting for a fix and switching to a new version of CMake is less evil than bypassing the problem through CMAKE_RC_COMPILE_OBJECT.

Reply:
Why does it look for `version.rc.res` in the first place? It should be just `version.res`...

Reply:
code:
```
message(STATUS " !!! inventory variable MSYS2_ARG_CONV_EXCL is: [$ENV{MSYS2_ARG_CONV_EXCL}] mast containing [/implib:] see https://www.msys2.org/docs/
```

Warns you to set the environment variable for msys2 MSYS2_ARG_CONV_EXCL=/implib:
In the opposite case build  abort with error:
```
lld-link: error: could not open 'C:\TOOLSET\msys64\implib;..\lib\clang_x64_lib\wxrc.lib': no such file or directory
```
Clang convert nix <> MS paths and options such as.  /implib:../myLib.lib   with msys2 help:  https://github.com/llvm/llvm-project/issues/61997 and  https://www.msys2.org/docs/

Reply:
If it's really a general problem with msys2 and clang, i.e. if it affects all projects, I don't think we should solve it at wx level. If it only affects wx, then I'd like to understand why, i.e. what are we doing wrong to run into this problem if the other projects don't suffer from it.

Should we actually use `--out-implib` or something like this instead of `/implib`? Where/whom are we passing this option to?

Reply:
```
Why does it look for version.rc.res in the first place? It should be just version.res...
```

I'm not a compiler connoisseur, but I've always seen object files my_file.cpp  as my_file.cpp.obj and my_resource.rc as my_resource.rc.res  ...


```
d:\source\contribution\wxWidgets\base\temp\build>dir "libs/base/CMakeFiles/wxbase.dir/__/__/__/__/src/msw"
 Volume in drive D is data
 Volume Serial Number is 0431-CC2F

 Directory of d:\source\contribution\wxWidgets\base\temp\build\libs\base\CMakeFiles\wxbase.dir\__\__\__\__\src\msw

19.04.2023  13:23    <DIR>          .
19.04.2023  13:23    <DIR>          ..
19.04.2023  13:22         4 017 425 basemsw.cpp.obj
19.04.2023  13:22            26 856 basemsw.cpp.obj.d
...

19.04.2023  13:22         4 698 283 utilsexc.cpp.obj
19.04.2023  13:22            27 227 utilsexc.cpp.obj.d
19.04.2023  13:23             1 020 version.rc.res.res.      
19.04.2023  13:23         4 315 976 volume.cpp.obj
19.04.2023  13:23            28 287 volume.cpp.obj.d
              47 File(s)     99 572 097 bytes
               2 Dir(s)  60 025 364 480 bytes free
```

Reply:
".rc.res.res"  exactly  is common problem in all our projects. Not wxWidgets only. 

Of course, we workaround  this problem by uses wxWidjets as 
```
...
include(./bugs/fixBefore.cmake)
add_subdirectory("wxWidjets")
include(./bugs/fixAfter.cmake)
...

```